code/modules/mob/living/carbon/xenomorph/castes/defiler/abilities_defiler.dm

```diff
@@ -129,8 +129,10 @@
 					continue
 				else
 					neuro_applied = TRUE
-			defile_strength_multiplier *= 2
```

Comment:
Did you remove this on purpose? It seems outside scope.

Reply:
Nope. Whoops.